### DISCUSSION TITLE
Do not overwrite default system DNS

### DISCUSSION THREAD

**Discussion by trenutoo** [pelican-dev/panel#128] [Ideas]
Wings forcing custom DNS nameservers causes more harm than good, leading to DNS issues that plague Pterodactyl users, for example, during server installations when ISP/host/user has restricted the available DNS.

The containers should inherit the system's or Docker's DNS configuration. If there's a problem with the system DNS, it would indicate a more severe issue with the system configuration that Wings shouldn't attempt to resolve.

**Comment by parkervcp** [comment]
I will have to spin up a few VM's with different default OS installs to see if I can find a good way to get the hosts DNS entries

**Reply by notAreYouScared** [reply]
@parkervcp Any update on this?

**Reply by parkervcp** [reply]
@notAreYouScared  I totally forgot to check on this. I have spun up a few VM's to check their defaults. I think covers the majority of defaults you may find online.

### use resolvectl
* Ubuntu 24.04
* Fedora 40

### use /etc/resolv.conf
* Alpine 3.20
* RHEL/Rocky 9.4
* debian 12.6
